src/helper/icon-by-entity-type.ts

```diff
@@ -47,6 +48,7 @@ export function getIconByTypename(
 
 export const entityIconMapping = {
   applet: faCubes,
+  audio: faPlay,
```

Comment:
plugins are not entities, so this is not the place.
also it seems unused, right?

Reply:
I see. So the usage in the privacy wrapper is not entirely correct but worked since videos and geogebra are also entities. That's also why it's confusing I guess ^^

Reply:
Thanks for checking.